### ISSUE TITLE
Support for other predicates besides 'cont'

### ISSUE THREAD

**Issue by apohllo** [kaspernj/simple_form_ransack#9]
I tried to use the gem with the following attribute:
``` 
f.input :surname_start
```
But I get an error: `Couldn't figure out attribute name from: surname_start`.
Running with Rails v. 4.2.1. 
The method works with the original Ransack helper.

**Comment by kaspernj** [comment]
I have fixed the issue in the following branch: "kj/start_end_support_on_attributes".

Please test it out and get back to me by altering your Gemfile like so:
```ruby
gem "simple_form_ransack", github: "kaspernj/simple_form_ransack", branch: "kj/start_end_support_on_attributes"
```

Looking forward to hearing back from you :-)

**Comment by apohllo** [comment]
I can confirm that the `start` predicate works as expected. Thank you very much indeed :-)

**Comment by kaspernj** [comment]
@apohllo I have released a new version with number 0.0.9 that you can use instead, since the branch have been deleted after the fix has been merged.

**Comment by apohllo** [comment]
Ok, thanks for info.
